db/config.cc

```diff
@@ -701,6 +701,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
             "Maximum number of distinct clustering key restrictions per query. This limit places a bound on the size of IN tuples, "
             "especially when multiple clustering key columns have IN restrictions. Increasing this value can result in server instability.")
     , alternator_port(this, "alternator_port", value_status::Used, 0, "Alternator API port")
+    , alternator_https_port(this, "alternator_tls_port", value_status::Used, 0, "Alternator API HTTPS port")
```

Comment:
You didn't rename the actual name of this option, just the variable name - I guess this wasn't deliberate?

Reply:
ugh, yes, a typo. New version coming soon

Reply:
done